assets/built-in/transformers/dockerfilegenerator/dotnetcore/templates/Dockerfile

```diff
@@ -12,32 +12,50 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM mcr.microsoft.com/dotnet/sdk:{{ .DotNetVersion }} AS builder
+{{ if .IncludeBuildStage }}
+
+# Build Stage
+FROM mcr.microsoft.com/dotnet/sdk:{{ .BuildStageImageTag }} AS {{ .BuildContainerName }}
+
+{{- if .IsNodeJSProject }}
+RUN apt-get update -y && \
+    apt-get install -y xz-utils && \
+    wget https://nodejs.org/dist/v16.16.0/node-v16.16.0-linux-x64.tar.xz && \
```

Comment:
Shouldn't this be dynamic?

Reply:
After this PR is merged https://github.com/konveyor/move2kube/pull/834 we can use the common logic.